Tuist/ProjectDescriptionHelpers/Target+Initializing.swift

```diff
@@ -48,19 +74,51 @@ extension Target {
         )
     }
 
-    public static func kifUITestsTarget(name: String, bundleId: String) -> Target {
+    fileprivate static func kifUITestsTarget(name: String, bundleId: String) -> Target {
         let targetName = "\(name)KIFUITests"
         return Target(
             name: targetName,
             platform: .iOS,
             product: .unitTests,
             bundleId: bundleId,
-            infoPlist: "\(targetName)/\(plistsPath)/Info.plist",
+            infoPlist: "\(targetName)/\(Constant.plistsPath)/Info.plist",
             sources: ["\(targetName)/**"],
             resources: [
                 "\(targetName)/**/.gitkeep", // To include empty folders
-            ], 
+            ],
             dependencies: [.target(name: name)]
         )
     }
 }
+
+// MARK: - Dependencies
+
+extension Target {
+
+    fileprivate static func makeFramework(module: Module, bundleId: String) -> [Target] {
```

Comment:
To consistent with other functions

```suggestion
    fileprivate static func frameworkTargets(module: Module, bundleId: String) -> [Target] {
```

Reply:
[a732c17](https://github.com/nimblehq/ios-templates/pull/536/commits/a732c17059fec3d44aac47494458332c22c60582)